app/models/plan.rb

```diff
@@ -62,6 +62,12 @@ def complete?
     state == :complete
   end
 
+  def required_signoffs
+    signoffs.build(user_id: user_id) if signoffs.find_by(user_id: user_id).blank?
```

Comment:
On a new job plan we don't have the user_id at this point so it defaults to the first user in the list...